cpp/CMakeLists.txt

```diff
@@ -754,10 +759,32 @@ if(CUDF_BUILD_TESTUTIL)
     cudftestutil PUBLIC "$<BUILD_INTERFACE:${CUDF_SOURCE_DIR}>"
                         "$<BUILD_INTERFACE:${CUDF_SOURCE_DIR}/src>"
   )
-
   add_library(cudf::cudftestutil ALIAS cudftestutil)
 
 endif()
+
+# * build cudf_identify_stream_usage --------------------------------------------------------------
+
+if(CUDF_BUILD_STREAMS_TEST_UTIL)
+  if(CUDA_STATIC_RUNTIME)
+    # TODO: Should this be a hard error, or just a skip? Depends on what the default is IMO.
```

Comment:
If the default is ON, then we should skip gracefully instead of issuing a fatal error.

Reply:
I decided to have `CUDF_BUILD_STREAMS_TEST_UTIL` default to the value of `CUDA_STATIC_RUNTIME`. I think that should be sufficient.